test-schema.json

```diff
@@ -27,6 +27,26 @@
                 "type": "array",
                 "items": { "$ref": "#/$defs/test" },
                 "minItems": 1
+            },
+            "specification":{
+                "description": "A reference to a specification document which defines the behavior tested by this test case. Typically this should be a JSON Schema specification document, though in cases where the JSON Schema specification points to another RFC it should contain *both* the portion of the JSON Schema specification which indicates what RFC (and section) to follow as *well* as information on where in that specification the behavior is specified.",
+
+                "type": "array",
+                "minItems": 1,
+                "items":{
+                    "properties": {
+                        "core": { "pattern": "^[0-9a-zA-Z]+(\\.[0-9a-zA-Z]+)*$", "type":"string" },
```

Comment:
Can you add metadata to each one of these (i.e. here in the schema some information about what this specification is, and probably a link to where it can be read).

Reply:
Sure, I'll be using description, right? or title?